libs/dapp-config/src/anchors/anchor-config.ts

```diff
@@ -10,15 +10,19 @@ import { calculateTypedChainId, ChainType } from '@webb-tools/sdk-core';
 export const anchorsConfig: ApiConfig['anchors'] = {
   [CurrencyId.webbETH]: {
     [PresetTypedChainId.ArbitrumTestnet]:
-      '0xa1a2b7e08793b3033122b83cbee56726678588b5',
-    [PresetTypedChainId.Goerli]: '0xa1a2b7e08793b3033122b83cbee56726678588b5',
-    [PresetTypedChainId.Sepolia]: '0xa1a2b7e08793b3033122b83cbee56726678588b5',
+      '0x12f721c568e907126e13d6664ebec606ee65ed3c',
+    [PresetTypedChainId.Goerli]: '0x12f721c568e907126e13d6664ebec606ee65ed3c',
+    [PresetTypedChainId.Sepolia]: '0x12f721c568e907126e13d6664ebec606ee65ed3c',
     [PresetTypedChainId.PolygonTestnet]:
-      '0xa1a2b7e08793b3033122b83cbee56726678588b5',
+      '0x12f721c568e907126e13d6664ebec606ee65ed3c',
     [PresetTypedChainId.OptimismTestnet]:
-      '0xa1a2b7e08793b3033122b83cbee56726678588b5',
+      '0x12f721c568e907126e13d6664ebec606ee65ed3c',
     [PresetTypedChainId.MoonbaseAlpha]:
-      '0xa1a2b7e08793b3033122b83cbee56726678588b5',
+      '0x12f721c568e907126e13d6664ebec606ee65ed3c',
+    [PresetTypedChainId.AvalancheFuji]:
+      '0x12f721c568e907126e13d6664ebec606ee65ed3c',
+    [PresetTypedChainId.ScrollAlpha]:
+      '0x12f721c568e907126e13d6664ebec606ee65ed3c',
   },
```

Comment:
@AtelyPham we do not want to replace these addresses rather we want to include the. Specifically, `0xa1a2b7e08793b3033122b83cbee56726678588b5` is the anchor address for the mocked webb bridge and `0x12f721c568e907126e13d6664ebec606ee65ed3c` is another webb based bridge. Collectively, the dApp would then support two distinct bridges with two distinct webb bridged assets.

Reply:
Okay, so I'll need to update the logic to be able to select multiple bridges. Do we have the design for that? @dutterbutter